dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx

```diff
@@ -456,7 +463,7 @@ const DesignableBannerV2: ReactComponent<BannerRenderProps> = ({
 						</div>
 					</div>
 				)}
-
+				{/* TODO: investigate if there is a reason why this is logically at the end of the grid rather than the beginning */}
```

Comment:
I just had a play with this and it appears to work the same if you move this to the start